### DISCUSSION TITLE
Scrolling/navigating inside interactive console

### DISCUSSION THREAD

**Discussion by j-fdion** [vercel/turborepo#8490] [Help]
### Summary

Hi! I wasn't sure where I could ask that...

Is is possible to scroll inside the interactive console?

The console displays: "Use arrow keys to navigate. Press `Enter` to interact with a task and `Ctrl-Z` to stop interacting"

I press enter and the console border goes yellow, but using arrow keys doesn't seem to do anything...

### Additional information

```js
package.json:
`"devDependencies": {
    "@acme/prettier-config": "workspace:*",
    "@turbo/gen": "^2.0.4",
    "prettier": "^3.3.1",
    "turbo": "^2.0.4",
    "typescript": "5.5.1-rc"
  }`
turbo.json:
    "dev:nextjs": {
      "persistent": true,
      "interactive": true,
      "cache": false
    },

The command I use:
"dev:next": "turbo watch dev -F @acme/nextjs",
```

**Comment by versecafe** [comment]
You can navigate between process with the arrow keys and scroll inside the process view using your mouse like you can in a standard terminal window *(assuming your emulator supports mouse based scrolling)*

**Reply by versecafe** [reply]
> Note you don't need to click the process just hover over and scroll on macOS, use your platforms equivalent to that and it should work fine

**Reply by j-fdion** [reply]
I'm using vscode with powershell terminal on windows, scrolling just scroll the whole terminal and not what's inside the yellow box, is there a config I must add somewhere? 

Thanks again.

**Reply by jonasrht** [reply]
Same here, did you find a solution for this?

**Reply by versecafe** [reply]
that might just be a vscode issue with it technically being a browser and some scroll behaviour issues, if extensions can modify terminal CSS then that might be the source of your issue

**Reply by brunobely** [reply]
Is there a way to change the keybinds? `u/d` for scrolling instead of `j/k` is absolutely diabolical.
